CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+## Bug Reporting :bug:
+
+One of the most important things you can do is report bugs. Please reference [how to report a bug](http://polite.technology/reportabug.html) and follow the issue templates when adding new bugs.
+
+## Development
+
+## Structure of the mlvet repository
+
+```
+mlvet
```

Comment:
Is this not the directory structure of like, every repository ever? Or have you just not filled out the template yet 😛

Reply:
Template is yet to be filled out :P